src/daft-scan/src/lib.rs

```diff
@@ -453,6 +508,8 @@ impl ScanTask {
                         }
                         #[cfg(feature = "python")]
                         FileFormatConfig::Database(_) => 0.0,
+                        #[cfg(feature = "python")]
+                        FileFormatConfig::PythonFunction => 0.0,
```

Comment:
Perhaps I should do `unreachable` instead? I think we should perform some refactors to group our file-based scans together, and split our Database/Python scans out to avoid this messy code.

Reply:
oh these probably shouldn't be 0.0 but instead 1.0 if they don't inflate